Makefile

```diff
@@ -122,6 +122,7 @@ server-test-mysql: export FB_STORE_TEST_DOCKER_PORT=44445
 
 server-test-mysql: ## Run server tests using mysql
 	@echo Starting docker container for mysql
+	docker-compose -f ./docker-testing/docker-compose-mysql.yml down -v --remove-orphans
```

Comment:
This change is to clean up after a previous test run crashed and left the container behind.